### DISCUSSION TITLE
[FR] Invert volume button page turning in landcape

### DISCUSSION THREAD

**Discussion by tarkeez** [plateaukao/einkbro#322] [Ideas]
I use my device in landscape mode with the volume rocker below the display, so vol- is left and vol+ is right facing, which makes page turning/scrolling a bit awkward since it's reversed.

It would be nice to have the option invert the rocker in landscape, so that vol+ (right side) would be next page/down and vol- (left side) would be previous page/up.

PS: My device doesn't report input events normally or pass them through the accessibility service, so I'm unable to use a third party button mapper.  Luckily KOReader is mappable. EinkBro is the only other app I would need this for.

**Comment by plateaukao** [comment]
You can lick on gesture button in menu, and  check "switch up/down direction"

**Comment by tarkeez** [comment]
Unfortunately this doesn't seem to work for the physical volume buttons, only the virtual touch buttons on screen.

Tried on my phone as well as the ereader.
